onnxruntime/core/providers/dnnl/subgraph/dnnl_conv.cc

```diff
@@ -21,13 +21,13 @@ void DnnlConv::CreatePrimitive(DnnlSubgraphPrimitive& sp, DnnlNode& node) {
 
   auto conv_src_mem = sp.GetMemory(node.Input(IN_X));
   auto src_md = conv_src_mem.get_desc();
-  src_md.data.format_kind = dnnl_format_kind_t::dnnl_format_kind_any;
-  auto src_dims = conv_src_mem.get_desc().dims();
+  src_md = dnnl::memory::desc(src_md.get_dims(), src_md.get_data_type(), dnnl::memory::format_tag::any);
```

Comment:
in the original code this memory descriptor was forced to be format_tag::any so that oneDNN chould choose the most optimal input memory shapes. This had a 10x impact on performance. Have you checked this still give the correct performance?

Reply:
I'm gathering results, but should we expect any difference? the new src_md is exactly the same as before, the only change is that since the object is now opaque we cannot directly change it's format as we did before, so instead we just create a new one with the same dimensions, data type and desired tag (_dnnl::memory::format_tag::any_)